content/version/1/4/code-of-conduct.id.md

```diff
@@ -0,0 +1,68 @@
++++
+version = "1.4"
+aliases = ["/version/1/4"]
++++
+
+# Kode Etik Contributor Covenant
+
+## Janji Kami
+
+Dalam minat kami untuk membuat lingkungan yang terbuka dan ramah, kami sebagai 
+kontributor dan pengelola berjanji untuk membuat partisipasi dalam proyek kami
+dan komunitas kami bebas dari pelecehan untuk semua orang, tanap memandang usia,
```

Comment:
'...dan komunitas kami bebas dari pelecehan untuk semua orang.'. you had a typo here: 'tanpa'.